pyaedt/hfss.py

```diff
@@ -300,8 +303,13 @@ def assigncoating(self, obj, mat=None,
 
         Examples
         --------
-        >>> id1 = aedtapp.modeler.primitives.get_obj_id("inner")
-        >>> coat = aedtapp.assigncoating([id1], "copper",usethickness=True, thickness="0.2mm")
+
```

Comment:
Good example!

Since this is a fairly long example, explain using plain english what you're doing.  For example:

Create a cylinder at the XY working plane and assign a copper coating of 0.2 mm to it.

Also, we only need one line of white space prior to the end of the docstring.